packages/postgres-database/src/simcore_postgres_database/migration/versions/1447a4f5d72b_add_product_s_group_column.py

```diff
@@ -0,0 +1,43 @@
+"""add product's group column
+
+Revision ID: 1447a4f5d72b
+Revises: fe19b4e93586
+Create Date: 2022-12-19 13:44:58.029875+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "1447a4f5d72b"
+down_revision = "fe19b4e93586"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("products", sa.Column("group_id", sa.BigInteger(), nullable=True))
```

Comment:
I do think we have ```gid``` everywhere don't we? maybe that would be nice to keep it the same for consistency?

Reply:
I never though i would say this but ... I would rather do the opposite, i.e. convert all ``gid`` into ``group_id``. I had a really bad time trying to match the indentifiers when i implemented the tags. My proposal:

- groups.id -> should be group's identifer in groups
- group_to